SmartDeviceLink/SDLManager.h

```diff
@@ -84,11 +84,6 @@ typedef void (^SDLManagerReadyBlock)(BOOL success, NSError *_Nullable error);
  */
 @property (strong, nonatomic, readonly, nullable) SDLRegisterAppInterfaceResponse *registerResponse;
 
-/**
```

Comment:
This should remain (even though it will always be `nil`) because the library should support the protocol. Android has the same accessor method in their library: https://github.com/smartdevicelink/sdl_android/blob/6f05f2d8cf18aa43369806d1e86b8ce90b144976/android/sdl_android/src/main/java/com/smartdevicelink/managers/SdlManager.java#L384

Reply:
Core said that it will never work though. It would be better to add it when it's actually functional, otherwise it's just confusing. @joeygrover